apps/src/gamelab/AnimationPicker/animationPickerModule.js

```diff
@@ -159,6 +159,23 @@ export function handleUploadError(status) {
 }
 
 /**
+ * The user chose to draw their own animation.  This concludes our picking
+ * process.  Dispatch action to add a new image, and then close the animation
+ * picker.
+ */
+export function pickNewAnimation() {
+  return (dispatch, getState) => {
+    const goal = getState().animationPicker.goal;
+    if (goal === Goal.NEW_ANIMATION) {
+      dispatch(addBlankAnimation());
+    } else if (goal === Goal.NEW_FRAME) {
+      // TODO (bbuchanan): Implement after integrating Piskel
```

Comment:
Hmm... should update these todos to say "Implement after attaching Piskel 'new frame' button to the animation picker."  Alternatively, I wonder if this will really ever be necessary?